src/Component/Categories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { statusCheck } from '../redux/categories/categories';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  const statusCheckEventHandler = () => {
+    dispatch(statusCheck());
+  };
+
+  const status = useSelector((state) => state.categories);
+
+  return (
+    <>
+      <button type="button" onClick={statusCheckEventHandler}>
+        Check status
+      </button>
+      <div>{status}</div>
+    </>
+  );
+};
+
+export default Categories;
```

Comment:
- [ ] When I click the categories link on your navbar, it doesn't route to the correct page. This is because you are linking to `'/categories'` while your element routes to `"/Bookstore/PageCatego"`. Kindly fix it. 👍

Reply:
Hi @Gambit142 ,

I do not understand this, from my end i can access the categories link on my navbar.
![image](https://user-images.githubusercontent.com/97225349/172502711-5d4ee2c4-fdd5-4443-996d-e46c0eca8afe.png)
Please could explain this in another way